src/main/java/controller/OrderController.java

```diff
@@ -0,0 +1,287 @@
+package controller;
+
+import exception.CarRentalException;
+import lombok.RequiredArgsConstructor;
+import models.cars.Car;
+import models.order.Order;
+import models.people.User;
+import models.session.Session;
+import services.car.CarService;
+import services.order.OrderService;
+import services.session.SessionService;
+import services.user.UserService;
+import util.DateTimeUtil;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static exception.ExceptionHandler.handleException;
+
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+    private final CarService carService;
+    private final UserService userService;
+    private final SessionService sessionService;
+    private final Scanner scanner;
+
+
+    public void programInterface() {
+        handleException(() -> {
+            Session session = sessionService.getActive();
+            if (userService.isManager(session.getUserId())) {
+                managerInterface();
+            } else {
+                userInterface();
+            }
+        });
+    }
+
+
+    private void managerInterface() {
+        while (true) {
+            if (sessionService.isUserAuthenticated()) {
+                System.out.println("\nChoose item:\n1 - Form order\n" +
```

Comment:
Maybe you need to check if user authenticated before getting the session

Reply:
Don't understand what you mean...